modules/manage/pages/security/encryption.adoc

```diff
@@ -371,16 +392,17 @@ See also: xref:./listener-configuration.adoc[Configure Listeners]
 
 == Use rpk with TLS
 
-If you're using `rpk` to interact with the Kafka API using mTLS identity (for example, to manage topics or messages), pass the `--tls-key`, `--tls-cert`, and `--tls-truststore` flags to authenticate.
+If you're using `rpk` to interact with the Kafka API using mTLS identity (for example, to manage topics or messages), pass the `--tls-key`, `--tls-cert`, and `--tls-truststore` flags to authenticate. If you have specified a certificate revocation list (`crl_file`) in your configuration, you can also use --crl-key`.
 
-To interact with the Admin API (for example, to manage users), pass the `--admin-api-tls-key`, `--admin-api-tls-cert`, and `--admin-api-tls-truststore` flags.
+To interact with the Admin API (for example, to manage users), pass the `--admin-api-tls-key`, `--admin-api-tls-cert`, `--admin-api-tls-truststore` and `--admin-api-tls-crl` flags.
```

Comment:
Does RPK support CRL files?

Reply:
Sorry @michael-redpanda, I should have highlighted this addition, which was more a question from side. I've asked the question in the #discuss-devex but have removed the new content until/unless I get a confirmation. Is that OK?